File: FrontEnd-ProjectReact/my-react-app-inventory/src/types/approvalHistory.ts
import type { Employee } from "./employee";

export interface ApprovalHistory {
  id: number;
  moduleName: string;
  refNo: string;
  approvedBy?: string | null;
  dateApproved?: Date | null;
  remarks?: string | null;
  recStatus?: string | null;
  
  approvedByNavigation?: Employee | null;
}

File: BackEnd-ProjectAPI/Sample/Sample.Infrastructure/EntityFramework/ApprovalHistory.cs
using System;
using System.Collections.Generic;

namespace Sample.Infrastructure.EntityFramework
{
    public partial class ApprovalHistory
    {
        public int Id { get; set; }
        public string ModuleName { get; set; } = null!;
        public string RefNo { get; set; } = null!;
        public string? ApprovedBy { get; set; }
        public DateTime? DateApproved { get; set; }
        public string? Remarks { get; set; }
        public string? RecStatus { get; set; }

        public virtual Employee? ApprovedByNavigation { get; set; }
    }
}
